INFO     root:test_runner.py:22 用例ID:2  模块:用户管理  场景:查询失败  标题:未登录状态, 查询用户列表失败
INFO     root:analyse_case.py:30 解析请求数据，解析结果：{'method': 'get', 'url': 'http://192.168.64.3:8888/api/private/v1/users', 'headers': None, 'params': {'pagenum': 1, 'pagesize': 1}, 'data': None, 'json': None, 'files': None}
INFO     root:send_requests.py:15 发起http请求 请求结果：{"data":null,"meta":{"msg":"无效token","status":400}}
INFO     root:asserts.py:16 HTTP响应断言  预期结果(无效token) in 实际结果({"data":null,"meta":{"msg":"无效token","status":400}})